### PR TITLE
polyg_8_lbar

### LINKED ISSUE #225: polyg_8_lbar
Description: An example similar to [polyg_8](https://geocat-examples.readthedocs.io/en/latest/gallery/Polygons/NCL_polyg_8.html#sphx-glr-gallery-polygons-ncl-polyg-8-py) except a color bar is used instead of a legend.

NCL Script: http://www.ncl.ucar.edu/Applications/Scripts/polyg_8_lbar.ncl
NCL Plot: http://www.ncl.ucar.edu/Applications/Images/polyg_8_lbar_lg.png

Category: Polygon

### PR DESCRIPTION
**Pull request by hCraker**
Closes #225 
NCL Plot: http://www.ncl.ucar.edu/Applications/Images/polyg_8_lbar_lg.png
image

### CODE REVIEW

**Comment by michaelavs** [comment]
> Good start so far! No real changes to the main polyg_8 code, but question about the box_1 code included in this repo; is this part of the PR or will it need to be deleted?



> Good start so far! No real changes to the main polyg_8 code, but question about the box_1 code included in this repo; is this part of the PR or will it need to be deleted?

Hold off on this comment, there appears box_1 is creating some issues with PRs

**Comment by michaelavs** [comment]
We went through this issue today in a group meeting. The issue should be resolved by using `git rebase`. This should get everything up-to-date on all branches which is most likely from a "sticky" file (box_1). The full work flow (for PRs) will now look like: `git checkout -b new_feature 
git push --set-upstream origin new_feature
<make changes>
git add files
**git pull**
git commit 
git push`

File: Plots/Polygons/NCL_polyg_8_lbar.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
+"""
+NCL_polyg_8.py
```
Comment (anissa111) [inline Plots/Polygons/NCL_polyg_8_lbar.py:2]:
```suggestion
NCL_polyg_8_lbar.py
```

File: Plots/Polygons/NCL_polyg_8_lbar.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+==============
+This script illustrates the following concepts:
+   - Drawing a scatter plot on a map
+   - Changing the marker color and size in a map plot
+   - Plotting station locations using markers
+   - Create a custom color bar
```
Comment (anissa111) [inline Plots/Polygons/NCL_polyg_8_lbar.py:8]:
```suggestion
   - Creating a custom color bar
```

**Review by anissa111 — APPROVED** [review]
This looks great! See minor comments below.

**Review by michaelavs — APPROVED** [review]
Looks good!

**Review by jukent — APPROVED** [review]
Great work!

**Pull request merged**